yt/frontends/ytdata/data_structures.py

```diff
@@ -499,10 +499,12 @@ def _parse_parameter_file(self):
                    self.base_domain_right_edge) < 0.5 * dx
 
         elif self.data_type == "yt_frb":
-            self.domain_left_edge = \
-              np.concatenate([self.parameters["left_edge"], [0.]])
-            self.domain_right_edge = \
-              np.concatenate([self.parameters["right_edge"], [1.]])
```

Comment:
We do also have a `uconcatenate` function that could be used for this.

Reply:
Yeah, I know.  I just found it a bit clunky to ensure everything was in the same units.  My solution doesn't actually work, though, so I'll do it like that.